From 72480e2650cce2702199f949b1705ea0f0c2a20d Mon Sep 17 00:00:00 2001
From: Ferdinand Xu <cheng.a.xu@intel.com>
Date: Tue, 21 Jul 2015 01:42:08 -0400
Subject: [PATCH 0201/1164] HIVE-11314: Print "Execution completed
 successfully" as part of spark job info [Spark
 Branch](Ferdinand Xu, reviewed by Xuefu Zhang)

---
 .../hadoop/hive/ql/exec/spark/SparkTask.java       |    1 +
 1 file changed, 1 insertion(+)

diff --git a/ql/src/java/org/apache/hadoop/hive/ql/exec/spark/SparkTask.java b/ql/src/java/org/apache/hadoop/hive/ql/exec/spark/SparkTask.java
index c65cd02..b28ba89 100644
--- a/ql/src/java/org/apache/hadoop/hive/ql/exec/spark/SparkTask.java
+++ b/ql/src/java/org/apache/hadoop/hive/ql/exec/spark/SparkTask.java
@@ -114,6 +114,7 @@ public int execute(DriverContext driverContext) {
           LOG.info(String.format("=====Spark Job[%s] statistics=====", jobRef.getJobId()));
           logSparkStatistic(sparkStatistics);
         }
+        LOG.info("Execution completed successfully");
       } else if (rc == 2) { // Cancel job if the monitor found job submission timeout.
         jobRef.cancelJob();
       }
-- 
1.7.9.5

